pkg/proxy/ipvs/proxier.go

```diff
@@ -258,9 +258,10 @@ type Proxier struct {
 	gracefuldeleteManager *GracefulTerminationManager
 }
 
-// IPGetter helps get node network interface IP
+// IPGetter helps get node network interface IP and IPs bound to dummy interface
 type IPGetter interface {
 	NodeIPs() ([]net.IP, error)
+	BoundIPs() (sets.String, error)
```

Comment:
We have a [set type for net.IP](https://github.com/kubernetes/utils/blob/master/net/ipnet.go#L125), wondering if we should use that here.

Reply:
GetLocalAddresses returns sets.String so it was simpler
But I wonder if we shouldn't always use net.IP (same for ListBindAddress which is very similar but returns []String)
Maybe put this in backlog for refactoring and keep this PR focused on the feature?

Reply:
sure